client/knowledge_base_search/src/__test__/containers/SearchContainer/FeatureSelect.test.jsx

```diff
@@ -11,59 +11,73 @@ describe('<FeatureSelect />', () => {
+  });
+
+  let passagesTab;
+  let relevancyTab;
+  let highlightedFeature;
+
+  function renderWithProps(propSet) {
+    wrapper = shallow(<FeatureSelect {...propSet} />);
+    passagesTab = wrapper.find('.feature_select--list_button').at(0);
+    relevancyTab = wrapper.find('.feature_select--list_button').at(1);
+    highlightedFeature = wrapper.find('.feature_select--list_button--active');
+  }
 
   function getButtonClickEvent(button) {
     return { target: { value: button.props().value } };
   }
 
+  function pressTab(featureWrapper, tabNumber) {
+    const button = featureWrapper.find('.feature_select--list_button').at(tabNumber);
+    button.simulate('click', getButtonClickEvent(button));
+  }
+
   it('renders without crashing', () => {
     const div = document.createElement('div');
     ReactDOM.render(<FeatureSelect {...props} />, div);
   });
 
-  it('has the expected features', () => {
-    wrapper = shallow(<FeatureSelect {...props} />);
+  it('has two features', () => {
+    renderWithProps(props);
 
     expect(wrapper.find('.feature_select--list_item')).toHaveLength(2);
   });
 
   describe('when the selectedFeature is "Passage Search"', () => {
-    const propsWithPassage = Object.assign({}, props, {
-      selectedFeature: FeatureSelect.featureTypes.PASSAGES.value,
-    });
-
     beforeEach(() => {
-      wrapper = shallow(<FeatureSelect {...propsWithPassage} />);
+      renderWithProps(props);
     });
 
     it('has "Passage Search" highlighted', () => {
-      const highlightedFeature = wrapper.find('.feature_select--list_button--active');
-
       expect(highlightedFeature.text()).toEqual(FeatureSelect.featureTypes.PASSAGES.text);
     });
+    it('disables tab selection for passages only', () => {
+      expect(passagesTab.props().disabled).toBe(true);
+      expect(relevancyTab.props().disabled).toBe(false);
+    });
   });
 
   describe('when the selectedFeature is "Relevancy"', () => {
-    const propsWithRelevancy = Object.assign({}, props, {
-      selectedFeature: FeatureSelect.featureTypes.TRAINED.value,
-    });
-
     beforeEach(() => {
-      wrapper = shallow(<FeatureSelect {...propsWithRelevancy} />);
+      renderWithProps(propsWithRelevancy);
     });
 
     it('has "Relevancy" highlighted', () => {
-      const highlightedFeature = wrapper.find('.feature_select--list_button--active');
-
       expect(highlightedFeature.text()).toEqual(FeatureSelect.featureTypes.TRAINED.text);
     });
+    it('disables tab selection for relevance only', () => {
+      expect(passagesTab.props().disabled).toBe(false);
+      expect(relevancyTab.props().disabled).toBe(true);
+    });
   });
 
   describe('when the first feature is clicked', () => {
     beforeEach(() => {
-      wrapper = shallow(<FeatureSelect {...props} />);
-      const button = wrapper.find('.feature_select--list_button').at(0);
-      button.simulate('click', getButtonClickEvent(button));
+      renderWithProps(propsWithRelevancy);
```

Comment:
shouldn't this now do nothing? I think you are changing the behavior to not allow clicking on tabs given the starting state (`props`) thus the expectation should be that it does NOT "call onSelect with 'passages'" (following test). If you want to say "when second feature is clicked" -> "and then first feature is clicked" -> it "calls onSelect with 'passages'", that would be more appropriate. An alternative is instead labeling this as "when the second feature is selected" -> "and then the first feature is clicked" -> it "calls onSelect with 'passages'"